Text/Neo Nectar.txt

```diff
@@ -5850,16 +5850,14 @@ AUTO [Hand]: At the beginning of your ride phase, if your vanguard is grade 1 or
 global.UnitGrade[CardStat] = 2
 global.CardInClan[CardStat] = 9
 }
-global.PowerStat[CardStat] = 10000
-global.DefensePowerStat[CardStat] = 5000
 global.TokenSummoner[CardStat] = 6130
 global.TokenSummonerPosition[CardStat] = 'SHIFT'
 global.TokenSummoner2[CardStat] = 7202
 global.TokenSummoner2Text[CardStat] = "Which token do you want to call?"
 global.TokenSummoner2Button1[CardStat] = "5000"
 global.TokenSummoner2Button2[CardStat] = "10000"
-global.PowerStat[CardStat] = 8000
-global.DefensePowerStat[CardStat] = 10000
+global.PowerStat[CardStat] = 10000
```

Comment:
Doesn't really matter but I think token summoner usually comes after the power stat? Resolve this comment if you think that it's a pick

Reply:
Looks like it's the last 5 or so cards that have been putting the stats under token summoner. They have all been amended to have the stats before token summoner